### PR TITLE
Update vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md

### PR DESCRIPTION
**Pull request by kc0705**

### CODE REVIEW

File: shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md [hunk 1]
```diff
@@ -13,17 +13,16 @@ TODO: परिष्करणीयम्
...
 
-ईदृश ग्रस्थरत्न मुद्रणाय श्रीश्री ब्रह्मतस्त्र परकाल महादेशि केन्द्रस्थाना दनुगृहीत मेकं पुस्तकम्, तथा मैनू र् राजकीय कोशागारा देकम्, अन्य चीरङ्गपट्टणा वासि श्रीनि वासरङ्गभट्टा रै ्यरधिगतम्, रामनाधपुरावासि श्रीरामक्रतु नाथवम्भ्यै श्शठकोपाचार्यै स्सम्प्राप्तगुपरम्, एतचीळोळ चतुष्टयावलोकन पूर्वकं यथामति संशोधित०चदानेन, 
+ईदृश ग्रस्थरत्नमुद्रणाय श्रीश्रीब्रह्मतन्त्र परकालमहादेशिकेन्द्रस्थानादनुगृहीतमेकं पुस्तकं, तथा मैसूर् राजकीयकोशागारादेकम्, अन्यच्छ्रीरङ्गपट्टणावासि श्रीनिवासरङ्गभट्टार्यैरधिगतम्, रामनाधपुरावासि श्रीरामक्रतुनाथवंश्यैश्शठकोपाचार्यैस्सम्प्राप्तनुपरम्, एतच्छ्रीकोशचतुष्टयावलोकनपूर्वकं यथामति संशोधितञ्दासेन। 
 
-( क ) 
-
-अस्य ग्रस्थस्य प्रणेतारो श्री वङ्गिवं शेश्वरान्वय कलशप योराशि राशाशशाङ्काः श्रीपै कुण्णदीक्षिताचार्याः, अत एवास्य वैकुण वीक्षितीय मितिच प्रसिद्धिः, एते वैदिक सार्वभौम काला दर्वाचीना एन, तथापि शीष्टानुष्ठानयोग्यं बहवः प्रतिपा द्यन्त इति मैसूर् राजकीय मुजराय् सूपरिण्टॆण्डॆण्ट् प्र भूणा माज्ञाधार केण दा सेन मुद्रापि लेस्मिन् ग्रन्थे अक्षर विन्या सर्खालित्यादिदोष सद्भावेसि "वरङ्क णे नियच्छति "ति न्यायेन निर्मि तृरा गुणग्राहिण स्तत्र भवन्तो महा स्सहृदयास्साड रं क्षन्तु महळा स्त्रीति सविनयं साखुलिबन्धं प्रार्थये। 
+***marker***
```
Comment (vvasuki) [inline shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:24]: hmm - just replace with underscores (we're using markdown) -

____________________________

Comment (kc0705) [reply shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:24]: Done

Comment (vvasuki) [reply shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:24]: Remove the text "marker" as well ...

File: shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md [hunk 2]
```diff
@@ -13,17 +13,17 @@ TODO: परिष्करणीयम्
...
 
-( क ) 
+___marker___
 
-अस्य ग्रस्थस्य प्रणेतारो श्री वङ्गिवं शेश्वरान्वय कलशप योराशि राशाशशाङ्काः श्रीपै कुण्णदीक्षिताचार्याः, अत एवास्य वैकुण वीक्षितीय मितिच प्रसिद्धिः, एते वैदिक सार्वभौम काला दर्वाचीना एन, तथापि शीष्टानुष्ठानयोग्यं बहवः प्रतिपा द्यन्त इति मैसूर् राजकीय मुजराय् सूपरिण्टॆण्डॆण्ट् प्र भूणा माज्ञाधार केण दा सेन मुद्रापि लेस्मिन् ग्रन्थे अक्षर विन्या सर्खालित्यादिदोष सद्भावेसि "वरङ्क णे नियच्छति "ति न्यायेन निर्मि तृरा गुणग्राहिण स्तत्र भवन्तो महा स्सहृदयास्साड रं क्षन्तु महळा स्त्रीति सविनयं साखुलिबन्धं प्रार्थये। 
+अस्य ग्रस्थस्य प्रणेतारो श्री वङ्गिवंशेश्वरान्वय कलशपयोराशिराकाशशाङ्काः श्रीवैकुण्ठदीक्षिताचार्याःाः, अत एवास्य वैकुदीक्षितीय मितिच प्रसिद्धिः, एते वैदिक सार्वभौम काला दर्वाचीना एन, तथापि शीष्टानुष्ठानयोग्यं बहवः प्रतिपा द्यन्त इति मैसूर् राजकीय मुजराय् सूपरिण्टॆण्डॆण्ट् प्र भूणा माज्ञाधार केण दा सेन मुद्रापि लेस्मिन् ग्रन्थे अक्षर विन्या सर्खालित्यादिदोष सद्भावेसि "वरङ्क णे नियच्छति "ति न्यायेन निर्मि तृरा गुणग्राहिण स्तत्र भवन्तो महा स्सहृदयास्साड रं क्षन्तु महळा स्त्रीति सविनयं साखुलिबन्धं प्रार्थये।
```
Comment (vvasuki) [inline shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:26]: "वैकुदीक्षितीय मितिच" इति परिष्करणीयं खलु …

Comment (kc0705) [reply shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:26]: Addressed.

Comment (vvasuki) [reply shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:26]: "र्याःाः" इति दोषः

File: shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md [hunk 3]
```diff
@@ -32,46 +32,20 @@ TODO: परिष्करणीयम्
 |                                                                               |पुटसङ्ख्यः|
 | :---------------------------------------------------------------------------- | ----: |
 | आचारलक्षणम् | १ |
```
[the remark this answers is not in the document]
Comment (vvasuki) [reply shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md:34]: Clarified as below

> I have used | symbol as part of creating the table of contents.  It is a formatting character in MD.  You may please check the preview and let me know.

**Pull request merged**